### DISCUSSION TITLE
Restrictions due to organization being flagged

### DISCUSSION THREAD

**Discussion by pschmid0** [community/community#126585] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

Why did our organization get restricted in using GitHub?

It could have been a couple of reasons: an expired credit card or a user that was hacked over Memorial Day.  Either way, our company is dead in the water until the issue is reviewed and the status changed back.  I believe our VP is owner of the organization and has already contacted support.

How can issues like this be expedited? What are other users experiences?

**Comment by gil-son** [comment]
Hello, how are you? I feel for that situation, you need to verify it!!


Access the [Github Support](https://support.github.com/)

Click on My Ticket -> New Ticket -> Finally describe it

If I supported you consider to mark my answer. Good Luck!

**Comment by LiteBrite82** [comment]
Hi @pschmid0 

Thank you for your patience. The community is unable to assist with nor escalate your ticket, but we can assure you it is in the right place. It will be answered in the order it was received based on our team's resources and the amount of tickets ahead of yours. We appreciate your understanding!

As far as expediting, multiple tickets tend to add confusion. Depending on the account type your org uses, there may be other avenues of communications; for instance [GitHub Enterprise](https://docs.github.com/en/get-started/learning-about-github/githubs-plans#github-enterprise) accounts have "enterprise support" included - your admin should have that information.

**Reply by pschmid0** [reply]
I did not enter a ticket since it may have already been entered by someone else.  The message and link on the screen tells everyone to contact support.  Are these multiple requests to contact support adding to the confusion?  Why does GitHub ask for them if it would be confusing?  How do I know if a ticket has already been created for the organization by someone else?

**Reply by LiteBrite82** [reply]
All great questions! duplicate tickets add to the queue length and slow the overall response time for all tickets. I dont know of a way to check if a ticket has already been submitted. Do you have an internal message board to coordinate current efforts; Slack, Teams?

**Reply by LiteBrite82** [reply]
One thing we can, and have, done is flag this discussion as an "account issue". It provides data to identify larger community wide issues; answering the question: Is this an isolated issue or is there a bigger problem like a bug or a security problem.

**Reply by pschmid0** [reply]
Thanks for your response!  Although we do have Teams, and I have a couple of ticket numbers, I don't know how, or if I can, access them.  The account holder for the organization (my boss) has said that everything on his side has been done, but he doesn't know, or hasn't been told, why the organization was locked and is still locked.  I am going to visit him in his office to see if I can get more info.

**Reply by pschmid0** [reply]
I visited my boss, and we can see that our organization account was compromised. This may have resulted in the locking of the account.  This is problematic as changes need to be made to keys and passwords, but with the account locked, anything in a repository can't be deployed.  My boss is willing to upgrade our paid account to Enterprise but is nervous about doing that and making the problem worse.

So it does look like it could be a security problem, but, without communication from GitHub, we don't know if it was that or a billing issue.

**Reply by pschmid0** [reply]
Still silence from GitHub.  The organization is a paying account, yet no response.

**Comment by pschmid0** [comment]
Still no response from GitHub.  And we are a paying customer!

**Comment by pschmid0** [comment]
Still waiting to hear from Support.
Lesson learned: if you are hacked, expect GitHub to shut down your account for an undetermined amount of time, maybe indefinitely, with no help from support.

**Comment by 0SOLO1** [comment]
Response #2

**Comment by asmawi117** [comment]
I also got hacked. The support team finally replied to me after 114 days, suggesting that I contact them or forward the message using the email associated with my hacked account. I have not heard back from them since, and they also closed my ticket. The issue still hasn't been resolved.

**Reply by pschmid0** [reply]
Thanks for the reply.  We have contacted support from the owner of the account, the hacked account, and members of the team.  So far, only silence.  We have real work to do and can spend no more time waiting on GitHub.

**Comment by pschmid0** [comment]
Still no response.  Our programming team is meeting today to discuss how we move away from GitHub.  Any suggestions on the best approach?

**Reply by LiteBrite82** [reply, marked as answer]
This community is managed by GitHub employees, but it is not a support forum and there is no SLA or guarantee each post will get a response from a community manager or product leader.

That being said, we were able to escalate this incident for clarification with another team. We believe an account you may be associated with and are referring to was unflagged on June 18th. Users should have full access now.
We are closing this thread as resolved.
